pyproject.toml

```diff
@@ -81,8 +81,8 @@ dev-docs = [
 ]
 
 dev-fmt = [
+    "ruff==0.1.3",
```

Comment:
Is strict pinning the right way to go here? We've been trying to move away from that towards semver ranges, but for dev tools it's _probably_ fine to pin. Since `ruff` is in `0.`, I'd assume the range here would be `>=0.1.3,<0.2`.

Reply:
We should be getting PRs from @dependabot for every ruff update. That will also let us know when a `ruff` upgrade introduces something that breaks our linting and let us take action.